src/routes/index.js

```diff
@@ -5,13 +5,23 @@ import login from './login.js'
 import invitation from './invitation.js'
 import forgotPassword from './forgotPassword.js'
 
-export default() => {
-  const apiServer = createApiServer(() => {}, () => {})
+export default () => {
+  function errorHandler (e) {
+    return {
+      status: e.status,
+      error: {
+        name: e.name,
+        message: e.message
+      }
+    }
+  }
+
+  const apiServer = createApiServer(errorHandler, () => {})
 
   admins(apiServer)
   login(apiServer)
   invitation(apiServer)
   forgotPassword(apiServer)
 
-  return apiServer
+  return apiServer._expressServer
```

Comment:
I would expose the apiServer, not the express server on it. In your tests, you can use the _expressServer prop without any issues.